### PR TITLE
Feature/400 correct all fake credentials to credentials variables in robot (show how to use them) and correct paths to be got from environment variables

### PR DESCRIPTION
**Pull request by WiktoriaPrusik**

### CODE REVIEW

File: Automation Examples/070.md [hunk 1]
```diff
@@ -53,13 +58,15 @@ foreach ♥mail in ♥mails
...
+
+It is a good practice to store usernames and passwords in Credential Container. In order to add a credential, choose Tools/Credential Container..., click "Add" and in the "Key" field insert the name of your value, for example "mail:address". Remember to have a key containing the `:` sign. In the "Value" and "Retype value" fields, insert for example your mail address. To the same with a mail password.
 
-Make sure to provide your correct credentials.
+See below how to access those values using the `♥credential` special variable.
 
 ```G1ANT
 list.create text ♥path result ♥attachments
-mail.smtp host ‴imap.gmail.com‴ login ‴user@gmail.com‴ password ‴p@$$w0rD‴ from ‴robot@gmail.com‴ to ‴robotuser@g1ant.com‴ subject ‴Csv attachment‴ body ‴Hi, here's your mail info‴ attachments ♥attachments
+mail.smtp host ‴poczta.o2.pl‴ login ♥credential⟦mail:address⟧ password ♥credential⟦mail:password⟧ from ♥credential⟦mail:address⟧ to ‴robotuser@g1ant.com‴ subject ‴Csv attachment‴ body ‴Hi, here's your mail info‴ attachments ♥attachments
```
Comment (danieladamkaiser) [inline Automation Examples/070.md]: I would suggest to use more anonimized smtp host instead of "poczta.o2.pl"

File: Automation Examples/070.md [hunk 2]
```diff
@@ -79,5 +89,5 @@ foreach ♥mail in ♥mails
 end foreach
 
 list.create text ♥path result ♥attachments
-mail.smtp host ‴imap.gmail.com‴ login ‴user@gmail.com‴ password ‴p@$$w0rD‴ from ‴robot@gmail.com‴ to ‴robotuser@g1ant.com‴ subject ‴Csv attachment‴ body ‴Hi, here's your mail info‴ attachments ♥attachments
+mail.smtp host ‴poczta.o2.pl‴ login ♥credential⟦mail:address⟧ password ♥credential⟦mail:password⟧ from ♥credential⟦mail:address⟧ to ‴robotuser@g1ant.com‴ subject ‴Csv attachment‴ body ‴Hi, here's your mail info‴ attachments ♥attachments
```
Comment (danieladamkaiser) [inline Automation Examples/070.md]: Same here - if possible use anonimized smtp host

**Review by danieladamkaiser — CHANGES_REQUESTED** [review]

File: Automation Examples/070.md (hunk 1 above)
Comment (WiktoriaPrusik) [reply Automation Examples/070.md]: maybe another credential variable instead of a visible host?

**Review by przegwar — CHANGES_REQUESTED** [review]
1. Nie podobają mi się runscripty
2. Przed i powinno być serduszko
3. klikając link here dostaję 404
4. Jeżeli chodzi o używanie shortcutów to raz klawisze są pisane z dużej a raz z małej litery np. "HOME" i "enter"

**Review by danieladamkaiser — APPROVED** [review]

File: Automation Examples/070.md (hunk 1 above)
Comment (danieladamkaiser) [reply Automation Examples/070.md]: Sure. Just make sure people will be instructed what to put there

**Comment by WiktoriaPrusik** [comment]
> 1. Nie podobają mi się runscripty
> 2. Przed i powinno być serduszko
> 3. klikając link here dostaję 404
> 4. Jeżeli chodzi o używanie shortcutów to raz klawisze są pisane z dużej a raz z małej litery np. "HOME" i "enter"

@przegwar please use english language. 

1. This pr is about credentials and paths so you shouldn't review my runscripts. As I said in one of my previous prs, I use runscripts when it is impossible to use something else. If you believe it is possible, show me an example and I will correct it. 
2. Before what "i" should be a heart sign? Can you please put comments in my code?
3. Which link doesn't work for you?

File: Automation Examples/010.md [hunk 3]
```diff
@@ -56,7 +58,7 @@ This for loop will create an `i` variable which will be initialised with 2 and i
```
Comment (przegwar) [inline Automation Examples/010.md:58]: ♥i

File: Automation Examples/020.md [hunk 4]
```diff
@@ -1,31 +1,35 @@
-# Process
+# Process Description
 
 Open https://www.vat19.com/, go to fun & strange stuff and download first 20 products' information (name, price, image) and save it into excel file. Template is available [here](Products%20Template.xlsx).
```
Comment (przegwar) [inline Automation Examples/020.md:4]: here nie działa

**Review by przegwar — CHANGES_REQUESTED** [review]

**Comment by przegwar** [comment]
W sumie trzeba by przemyśleć jak zmienić addon selenium, żeby zminimalizować użycie runscriptów. Wymaganie znajomości js bardzo podnosi próg wejścia.

File: Automation Examples/020.md (hunk 4 above)
Comment (WiktoriaPrusik) [reply Automation Examples/020.md:4]: @przegwar a good practice is to use english in PRs...

Comment (WiktoriaPrusik) [reply Automation Examples/020.md:4]: and it works 😛 check it again: https://github.com/G1ANT-Robot/G1ANT.Tutorials/blob/34ea0b2f22cd2bb090fa6906094115a2d5236bbe/Automation%20Examples/020.md

**Comment by WiktoriaPrusik** [comment]
> W sumie trzeba by przemyśleć jak zmienić addon selenium, żeby zminimalizować użycie runscriptów. Wymaganie znajomości js bardzo podnosi próg wejścia.

So you agree that there isn't a runscript in my code that could be replaced? Cool.

File: Automation Examples/010.md (hunk 3 above)
Comment (WiktoriaPrusik) [reply Automation Examples/010.md:58]: Thanks for noticing that 👍

**Comment by WiktoriaPrusik** [comment]
Also @przegwar it is worth to mention that this is a public repo and there are people outside of our organisation that are interested in getting to know what we are doing, so English language is - I would say - crucial here. I have some followers and some of them follow our repos 😛

**Review by przegwar — APPROVED** [review]

**Pull request merged**